app/components/Icons/User.tsx

```diff
@@ -0,0 +1,22 @@
+export default function User(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
suggestion: avoid the default so that then you can create an index file  `src/components/Icons/index.ts`
```
export { User } from "./User";
export { Letter } from "./Letter";
export { Lock } from "./Lock";
export { LoadingSpinner } from "./LoadingSpinner";
export { Confetti } from "./Confetti";
```
and import all the icons at once `import { User, Letter, Lock, LoadingSpinner, Confetti } from "./Icons";`